### ISSUE TITLE
Поиск по документу

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#36]
```
Планируется ли добавление функционала по 
поиску внутри докумениа?
```

Original issue reported on code.google.com by `imr...@gmail.com` on 28 Aug 2011 at 6:26

**Comment by GoogleCodeExporter** [comment]
Original comment by `Alexander.V.Kasatkin@gmail.com` on 28 Aug 2011 at 7:01
- Changed state: **Discussion**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Планируется.Сначала реализация ссылок, 
потом поиск. 
```

Original comment by `mc.creat` on 28 Aug 2011 at 7:11

**Comment by GoogleCodeExporter** [comment]
Original comment by `mc.creat` on 29 Aug 2011 at 1:08
- Changed state: **Postponed**

**Comment by GoogleCodeExporter** [comment]
```
Туда же бы хотелось выделение\копирование 
текста.
```

Original comment by `imr...@gmail.com` on 3 Sep 2011 at 11:21

**Comment by GoogleCodeExporter** [comment]
```
За неимением адекватного outline-а внутренних 
(да и внешних) ссылок оченно не хватает ) 
P.S.:Как и копирования текста,поиска, 
переводчика... Эх,  размечтался... )
P.P.S.: Оромное спасибо за последний билд! 
Аппаратное ускорение и вправду работает 
(правда чуток с рывками)
```

Original comment by `imr...@gmail.com` on 12 Dec 2011 at 4:27

**Comment by GoogleCodeExporter** [comment]
Original comment by `Alexander.V.Kasatkin@gmail.com` on 20 Mar 2012 at 10:47
- Added labels: **search**

**Comment by GoogleCodeExporter** [comment]
```
Implemented in 1.5
```

Original comment by `Andrei.K...@gmail.com` on 3 May 2012 at 3:52
- Changed state: **Fixed**
